docs/src/tutorial/kubectl.md

```diff
@@ -0,0 +1,101 @@
+# Basic operations with Kubernetes using kubectl
+
+## What you will need
+
+Before you begin, make sure you have the following:
+
+- A bootstrapped Canonical K8s cluster (See [insert link]...)
+
+### 1. The Kubectl Command
+
+This commands interacts with the Kubernetes API server (kube-apiserver) and is the most commonly used command when working with Kubernetes, so let's take some time to familiarize ourselves with it.
```

Comment:
I would shorten this sentence to something like this, use an "active voice" and include a refernce:
Use the `kubectl` command to interact with your cluster, establishing a connection with the [Kubernetes API server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).